tekton/release-pipeline-run.yaml

```diff
@@ -11,9 +11,9 @@ spec:
   - name: versionTag
     value: v0.X.Y # REPLACE with the version you want to release, including `v`. For nightly releases, we might want to use `vYYYYMMDD-commit`
   - name: imageRegistry
-      value: gcr.io/tekton-releases # REPLACE with your own registry
-    - name: package
-      value: github.com/tektoncd/pipeline
+    value: gcr.io/tekton-releases # REPLACE with your own registry
+  - name: package
+    value: github.com/tektoncd/pipeline
```

Comment:
:thinking:

Reply:
yeah the YAML indentation is wrong : https://github.com/tektoncd/pipeline/blob/master/tekton/release-pipeline-run.yaml#L14